packages/types/src/node-protocol.ts

```diff
@@ -79,6 +79,9 @@ export namespace Node {
     timeout: BigNumber;
     initialState: AppState;
   };
+  export interface InterNodeProposeInstallParams extends ProposeInstallParams {
```

Comment:
@IIIIllllIIIIllllIIIIllllIIIIllllIIIIll curious your thoughts here

Reply:
Only reason this exists is because no interface existed for an initiating Node (who creates an AppInstanceId for a proposed AppInstance) to tell a peer Node about said proposal's AppInstanceId.

Not capturing this need via an interface would get hacky and brittle. I'm not married to this naming either but wanted to emphasize this is strictly a Node-to-Node interface concern.